Create mysql_performance_temp_ondisk_table_high.md (#57)
percona-platform/portal-doc: docs/advisors/checks/mysql_performance_temp_ondisk_table_high.md

> # mysql_performance_temp_ondisk_table_high

Author: @kedarvj, please suggest a human-friendly title here. Thanks!

> # mysql_performance_temp_ondisk_table_high

Author: ```suggestion
# Performance check for On-Disk Temporary MySQL tables
```

## Description
Temporary tables on disk are slower than in-memory tables since disk I/O operations are inherently slower than memory operations. This can be especially noticeable if the temporary table is being used heavily, as the frequent reads and writes to the disk can cause performance issues. 

> This can often be attributed to an unoptimized query or the absence of an index, among other factors and a query review or configuration change can help.

Reviewer: ```suggestion
This can often be attributed to an unoptimized query or the absence of an index, among other factors. In such situations a query review or configuration change can be useful.
```

## Resolution

> Perform a query review to analyze what queries are causing the temporary tables. Query review will help identify poorly written queries, table design issues or missing indexes and help optimize the queries causing temp disk tables.

Reviewer: ```suggestion
Perform a query review to identify which queries are causing the temporary tables.
Query review will help identify poorly written queries, table design issues or missing indexes, and will help optimize any queries that cause temporary disk tables.
```

> Review tmp_table_size and max_heap_table_size only when the query review isn’t yielding results and it is absolutely necessary to raise in memory temporary table types.

Reviewer: ```suggestion
Review **tmp_table_size** and **max_heap_table_size** only when the query review isn’t yielding results and it is absolutely necessary to raise in-memory temporary table types.
```
